manifests/plugin/rabbitmq.pp

```diff
@@ -0,0 +1,56 @@
+# rabbitmq plugin
+# https://pypi.python.org/pypi/collectd-rabbitmq
+#
+# == Class collectd::plugin::rabbitmq
+#
+#  Configures rabbitmq metrics collection. Optionally installs the plugin
+#  Note, it is up to you to support package installation and sources
+#
+# === Parameters:
+# [*ensure*]
+#   String
+#   Passed to package and collectd::plugin resources ( both )
+#   Default: present
+#
+# [*interval*]
+#   Integer
+#   Interval setting for the plugin
+#   Default: $::collectd::params::interval
+#
+# [*manage_package*]
+#   Boolean
+#   Toggles installation of plugin. Please reference https://collectd-rabbitmq.readthedocs.org/en/latest/installation.html
+#   Default: false
+#
+# [*package_provider*]
+#   String
+#   Passed to package resource
+#   Default: pip
+#
+class collectd::plugin::rabbitmq (
+  $config           = {
+    'user'     => 'guest',
+    'password' => 'guest',
+    'host'     => $::fqdn,
+    'port'     => 15672,
+    'scheme'   => 'http',
+  },
+  $ensure           = 'present',
+  $interval         = $::collectd::params::interval,
+  $manage_package   = false,
+  $package_provider = 'pip',
+) {
+
+  if $manage_package {
```

Comment:
Does this exist on Debian-ish as well? The rest of the plugins check for RedHat as an osfamily.